src/cryptoadvance/specter/helpers.py

```diff
@@ -273,3 +297,22 @@ def sort_descriptor(cli, descriptor, index=None, change=False):
         desc = f"{p}({desc})"
 
     return AddChecksum(desc)
+
+def hash_password(password):
+    """Hash a password for storing."""
+    salt = hashlib.sha256(os.urandom(60)).hexdigest().encode('ascii')
+    pwdhash = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), 
+                                salt, 10000)
```

Comment:
I suggest making a few changes here:
- store salt and password hash separately to avoid things like `stored_password[64:]`, so password key would contain `salt` and `pwdhash` keys.
- use `pbkdf2_hmac('sha256',...)` to make resulting hash x2 shorter, I don't see security benefits of sha512 here.
- store bytes not as hex digits but use base64 encoding (`binascii.b2a_base64` and `binascii.a2b_base64`) - again, shorter.

One note about `b2a_base64` function - it adds `\n` at the end of the string for some reason, but it can be removed using `.strip()` on it. It doesn't influence the result of `a2b_base64` but I am not sure how json will react to `\n` in the string